bin/prettier.js

```diff
@@ -33,15 +41,14 @@ if (!filenames.length && !stdin) {
   process.exit(1);
 }
 
+function formatWithShebang(input) {
+  const programInput = input.split("\n");
```

Comment:
Agreed, noticed this as well. Please look for the initial `\n` as stated above and slice it.